src/main/java/org/folio/gobi/LookupService.java

```diff
@@ -68,19 +69,26 @@ public LookupService(RestClient restClient) {
    * @param location
    * @return UUID of the location
    */
-  public CompletableFuture<String> lookupLocationId(String location) {
+  public CompletableFuture<LocationTranslationResult> lookupLocationId(String location) {
     logger.debug("lookupLocationId:: Trying to look up locationId by location '{}'", location);
-    String query = HelperUtils.encodeValue(String.format(CQL_CODE_STRING_FMT, location));
-    String endpoint = String.format(LOCATIONS_ENDPOINT + QUERY, query);
-    return restClient.handleGetRequest(endpoint).toCompletionStage().toCompletableFuture()
+    return restClient.handleGetRequest(LOCATIONS_ENDPOINT).toCompletionStage().toCompletableFuture()
```

Comment:
I suppose limit param will be 10 if we not specify it, we need to check what max limit is supported by mod-search and use it

Reply:
I see, 1000 is max
{
    "errors": [
        {
            "message": "getConsortiumLocations.limit must be less than or equal to 1000",
            "type": "ConstraintViolationException",
            "code": "validation_error",
            "parameters": []
        }
    ],
    "total_records": 1
}

Reply:
We need at least 10000, maybe if we not specify limit param than we will get all data, need to check it by inserting 10k records of locations

Reply:
looking into code of mod-search, if we not provide limit param - ES will set -1, so grab all data, but need to check via API call for example to fetch 2000k locations